### PR TITLE
Light Table Style and Table Summary

### PR DESCRIPTION
**Pull request by svanherk**
image

### CODE REVIEW

File: components/d2l-evaluation-hub/build/lang/ar.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const LangArImpl = (superClass) => class extends superClass {
 			'masterTeacher': 'Master Teacher',
 			'noSubmissions': 'There are no submissions that need your attention.',
 			'submissionDate': 'تاريخ الإرسال',
+			'tableTitle': 'List of unevaluated Learner submissions from across courses and tools',
```
Comment (mtjandra) [inline components/d2l-evaluation-hub/build/lang/ar.js]: Should `Learner` be lower case?

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 2]
```diff
@@ -87,7 +95,8 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 					@apply --d2l-body-compact-text;
 				}
 			</style>
-			<d2l-table hidden$="[[_fullListLoading]]" aria-colcount$="[[_headerColumns.length]]" aria-rowcount$="[[_data.length]]">
+			<d2l-offscreen id="d2l-evaluation-hub-activities-list-table-summary">[[localize('tableTitle')]]</d2l-offscreen>
+			<d2l-table type="light" hidden$="[[_fullListLoading]]" title="[[localize('tableTitle')]]" aria-describedby$="d2l-evaluation-hub-activities-list-table-summary" aria-colcount$="[[_headerColumns.length]]" aria-rowcount$="[[_data.length]]">
```
Comment (mtjandra) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Does `aria-describedby` need that `$`? Seems like some hardcoded property similar to `type=light`

File: components/d2l-evaluation-hub/build/lang/ar.js (hunk 1 above)
Comment (jeroach) [reply components/d2l-evaluation-hub/build/lang/ar.js]: nit: `noSubmissions` ends with a period. This term does not. Should we be consistent? Or are they used in different scenarios?

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 3]
```diff
@@ -26,7 +26,15 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 		const evaluationHubActivitiesListTemplate = html`
 			<style include="d2l-table-style">
 				d2l-td {
-					font-weight: normal;
+					font-size: 0.7rem;
+				}
+				d2l-td.d2l-username-column {
+					font-size: 0.8rem;
```
Comment (jeroach) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Why do we need a larger font size for the username column? Design? :)

File: components/d2l-evaluation-hub/build/lang/ar.js (hunk 1 above)
Comment (svanherk) [reply components/d2l-evaluation-hub/build/lang/ar.js]: This is the exact text Surekha had in the story, so I assumed there was reason for the uppercase.  You'll see this when you hover over the table, which is maybe why a period wasn't wanted.  @Surekh?

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js (hunk 3 above)
Comment (svanherk) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Yeah, this was Erin's request

[the remark this answers is not in the document]
Comment (svanherk) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Ah yes thanks!  Missed this when I was selecting stuff to stage

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js (hunk 2 above)
Comment (svanherk) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: So this is an attribute that screenreaders look for, not something that `d2l-table` has as a property.  Without the `$`, it won't show up.  Whereas `type` is a property on `d2l-table`.

File: components/d2l-evaluation-hub/build/lang/ar.js (hunk 1 above)
Comment (Surekh) [reply components/d2l-evaluation-hub/build/lang/ar.js]: Confirmed with Dan upper case for Learner is good.

**Pull request merged**